apps/site/tests/universal/page-header.test.ts

```diff
@@ -58,35 +58,23 @@ test("page header navigation works", async ({
   await expect(page).toHaveURL("/signup");
 });
 
-test("triggers for search modal work", async ({ page, isMobile }) => {
-  test.skip(!!isMobile, "This feature does not exist on mobile");
-
+test("search modal is triggered by button press on desktop", async ({
+  page,
+  isMobile,
+}) => {
   await page.goto("/");
 
-  const searchModalLocator = page.locator('[data-testid="bp-search-modal"]');
-
-  await expect(searchModalLocator).not.toBeVisible();
-  await page.keyboard.press("/");
```

Comment:
`/` logic was moved into a new test file that's focused on search modal. We can add more tests to it later.